modules/nw-ovn-kubernetes-migration-about.adoc

```diff
@@ -0,0 +1,103 @@
+
+The MCO updates a single machine per pool at a time by default, so the total time the migration takes increases with the size of the cluster.
+
+[id="considerations-migrating-ovn-kubernetes-network-provider_{context}"]
+== Considerations for migrating to the OVN-Kubernetes network provider
+
+While the OVN-Kubernetes network provider implements many of the capabilities present in the OpenShift SDN network provider, the configuration is not the same.
+If your cluster uses any of the following OpenShift SDN capabilities, you must manually configure the same capability in OVN-Kubernetes:
+
+* Egress IP addresses
+* Egress network policies
+* Egress router pods
+* Multicast
+* Network policies
+
+The following sections highlight the differences in configuration between the aforementioned capabilities in OVN-Kubernetes and OpenShift SDN.
+
+[discrete]
+=== Egress IP addresses
+
+The differences in configuring an egress IP address between OVN-Kubernetes and OpenShift SDN is described in the following table:
+
+.Differences in egress IP address configuration
+[cols="1a,1a",options="header"]
+|===
+|OVN-Kubernetes|OpenShift SDN
+
+|
+* Create an `EgressIPs` object
+* Add an annotation on a `Node` object
+
+|
+* Patch a `NetNamespace` object
+* Patch a `HostSubnet` object
+|===
+
+For more information on using egress IP addresses in OVN-Kubernetes, see "Configuring an egress IP address".
+
+[discrete]
+=== Egress network policies
+
+The difference in configuring an egress network policy, also known as an egress firewall, between OVN-Kubernetes and OpenShift SDN is described in the following table:
+
+.Differences in egress network policy configuration
+[cols="1a,1a",options="header"]
+|===
+|OVN-Kubernetes|OpenShift SDN
+
+|
+* Create an `EgressFirewall` object in a namespace
+
+|
+* Create an `EgressNetworkPolicy` object in a namespace
+|===
+
+For more information on using an egress firewall in OVN-Kubernetes, see "Configuring an egress firewall for a project".
+
+[discrete]
+=== Egress router pods
+
+OVN-Kubernetes does not support using egress router pods in {product-title} 4.6.
+
+[discrete]
+=== Multicast
+
+The difference in enabling multicast traffic on OVN-Kubernetes and OpenShift SDN is described in the following table:
+
+.Differences in multicast configuration
+[cols="1a,1a",options="header"]
+|===
+|OVN-Kubernetes|OpenShift SDN
+
+|
+* Add an annotation on a `Namespace` object
+
+|
+* Add an annotation on a `NetNamespace` object
+|===
+
+For more information on using an egress firewall in OVN-Kubernetes, see "Enabling multicast for a project".
+
+[discrete]
+=== Network policies
+
+OVN-Kubernetes fully supports the Kubernetes `NetworkPolicy` API in the `networking.k8s.io/v1` API group. No charges are necessary in your network policies when migrating from OpenShift SDN.
```

Comment:
No charges ?  typo ?

Reply:
By the way, the original suggestions for item 1 :All the openshift-sdn specific configuration cannot be migrated to OVN. It means specific configuration only existing in openshift-sdn cannot be migrated. Such as SDN multitenant mode, cannot be migrated to OVN.  Thanks!

Reply:
@huiran0826, oh, I forgot to mention OpenShift SDN isolation modes. I'll go back and add a note about that, too. Thanks!

Reply:
@huiran0826, I've added a section about namespace isolation as well.

Reply:
@jboxman I think there is some misunderstanding here. If sdn cluster is in multitenant mode, then it cannot be migrated to OVN directly.  That means only sdn is in network policy mode, then can do migration to OVN in this way.

Reply:
@huiran0826, ah, okay, so the cluster must be using OpenShift SDN in network policy mode or a migration is not possible at all?

Reply:
@jboxman You are right.

Reply:
@huiran0826 thanks!